### PR TITLE
[12.0][BACKPORT] l10n_ch_qriban

### PR DESCRIPTION
**Pull request by yvaucher**
This is a backport of module l10n_ch_qriban from odoo core 13.0.

This is required to implement qr_bill in version 12.0.

The new dependency tree is the following:

```
l10n_ch
└── l10n_ch_base_bank
    └── l10n_ch_fix_isr_reference
        ├── l10n_ch_payment_slip
        └── l10n_ch_qriban
```

### CODE REVIEW

File: l10n_ch_payment_slip/__manifest__.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
  'depends': [
      'account',
      'l10n_ch_base_bank',
+     'l10n_ch_fix_isr_number',
```
Comment (i-vyshnevska) [inline l10n_ch_payment_slip/__manifest__.py]:
```suggestion
     'l10n_ch_fix_isr_reference',
```

**Review by i-vyshnevska — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@yvaucher the bot didn't merge it. Do you launch it again?

**Comment by yvaucher** [comment]
/ocabot merge nobump

Let's try it

**Pull request merged**